src/api/student/student.service.ts

```diff
@@ -106,4 +107,112 @@ export class StudentService extends APIService<
 
     return studentPathInstance;
   }
+
+  async finishExams(user: Payload, createUserDto: FinishStudentQuiz): Promise<QuizInstance> {
+    const { fullMark, mark, answeredOptions, quizzId } = createUserDto;
+    const quizInstance = await this.db.quizInstance.findUnique({
```

Comment:
is there a reason why we are selecting quizInstance instead of QuizInstanceStudent?